components/form.js

```diff
@@ -737,19 +810,85 @@ export function InputUserSuggest ({
             autoComplete='off'
             onChange={(formik, e) => {
               onChange && onChange(formik, e)
+              if (e.target.value === ovalue) {
+                // we don't need to set the ovalue or query if the value is the same
+                return
+              }
```

Comment:
This is a fix that I made that's unrelated to this PR. It was a bug where a selecting a suggestion, would re-query because of a race between `onSelect` and `onChange`.